api/lib/application/pole-emplois/pole-emploi-controller.js

```diff
@@ -9,6 +10,7 @@ module.exports = {
     const { userId, idToken } = await usecases.createUserFromPoleEmploi({ authenticationKey });
 
     const accessToken = tokenService.createAccessTokenFromUser(userId, 'pole_emploi_connect');
+    await userRepository.updateLastLoggedAt({ userId });
```

Comment:
**Question :** C'est la PR aux questions 😅
J'ai noté que pour Pôle Emploi et GAR , dans l'ordre on créer l'access token puis on met à jour le `lastLoggedAt`.
Mais l'inverse est fait dans `authenticate-external-user` et `authenticate-user`.

Je me demande si l'ordre à une importance ou pas. Si pour une raison l'access token n'est pas créé ( cet incident peut-il arriver ?) on ne devrait pas mettre à jour le `lastLoggedAt` ? 

(je n'ai pas trouvé de gestion d'erreur dans le tokenService, je ne sais donc pas si ce cas peut survenir 🤔 )

Reply:
Ah bah je me suis posée la même question, mais en effet comme il n'y a pas d'erreur dans le tokenService je n'ai pas l'impression que l'ordre a d'importance. On peut quand même décider d'unifier ça en gardant un seul même ordre partout. Je n'ai pas de préférence là dessus :)

Reply:
vu ensemble : on met le saveLoggedAt avant la génération de token